dashboard/src/index.css

```
@@ -0,0 +1,94 @@
+.job-detail dd {
+  margin: 0;
+}
+
+.job-output {
+  background: rgba(128, 128, 128, 0.1);
+  padding: 0.75rem;
+  border-radius: 6px;
+  overflow-x: auto;
+  font-size: 0.85rem;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)